torchao/prototype/moe_training/kernels/mxfp8_blocked_scales.py

```diff
@@ -79,34 +92,44 @@ def torch_to_blocked_2d_K_groups(
     """
     assert x_scales.ndim == 2, "x_scales must be 2D"
     assert block_size == 32, "Only block_size=32 is supported for now"
-    blocked_scales_list = []
+    M, total_K = x_scales.shape
```

Comment:
Note for reviewers: the changes to the torch reference implementation `torch_to_blocked_2d_K_groups` are not working properly yet. It does not match the triton kernel in unit tests, but I am sure the triton kernel is correct because the e2e training tests validate correct gradient numerics. As further evidence, replacing the triton kernel with this torch impl in the training code, and the gradient numerics are garbage. 

Need to figure out the correct way to represent "row of blocks"-major PER GROUP in torch native code, before landing this.